libs/pages/layout/src/lib/feature/layout/layout.tsx

```diff
@@ -82,11 +100,19 @@ export function Layout(props: PropsWithChildren<LayoutProps>) {
     setCurrentProjectIdOnStorage(projectId)
   }, [organizationId, projectId])
 
-  useStatusWebSockets()
-
   return (
     <LayoutPage topBar={topBar} cluster={clusters[0]} defaultOrganizationId={organizations[0]?.id}>
       <>
+        {
+          /**
+           * XXX: Here we are limited by the websocket API which requires a clusterId
+           * We need to instantiate one hook per clusterId to get the complete environment statuses of the page
+           */
+          clusters.map(
+            ({ id }) =>
+              organizationId && <WebSocketListenerMemo key={id} organizationId={organizationId} clusterId={id} />
+          )
+        }
         <WebsocketContainer />
```

Comment:
Need to remove it ?

Reply:
No we can't because this is still used for the cluster list page